src/utils/finance/source.py

```diff
@@ -294,25 +326,74 @@ def _standardize_null_values(
     ) -> pd.DataFrame:
         """Replace implicit null values with actual null values"""
         for column_name, null_values in self.null_values.items():
+            if column_name not in standard_schema_table.columns:
+                continue
             for implicit_null_value in null_values:
                 standard_schema_table[column_name] = standard_schema_table[
                     column_name
                 ].replace(implicit_null_value, pd.NA)
         return standard_schema_table
 
-    def _filter_data(self, standard_schema_table: pd.DataFrame) -> pd.DataFrame:
-        """Filter data based on filter configuration"""
-        for column_name, filter_values in self.filter.items():
-            if column_name not in standard_schema_table.columns:
-                continue
-            if "NOT" in filter_values:
-                standard_schema_table = standard_schema_table[
-                    ~standard_schema_table[column_name].isin(filter_values["NOT"])
-                ]
+    def _standardize_transaction_direction(
+        self, standard_schema_table: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Flip donor and recipient columns if 'transaction_direction' is 'reverse'
+
+        In state config, if a raw column is mapped to transaction_direction and
+        the enum mapper maps some values to 'reverse' those rows will switch
+        donor and recipient columns.
+        """
+        if "transaction_direction" not in standard_schema_table.columns:
+            return standard_schema_table
+        reverse_mask = standard_schema_table["transaction_direction"] == "reverse"
+        reverse_table = standard_schema_table.loc[reverse_mask, :]
+        new_reverse_table_columns = []
+        for column in standard_schema_table.columns:
+            if column.startswith("donor"):
+                new_reverse_table_columns.append(column.replace("donor", "recipient"))
+            elif column.startswith("recipient"):
+                new_reverse_table_columns.append(column.replace("recipient", "donor"))
             else:
-                standard_schema_table = standard_schema_table[
-                    standard_schema_table[column_name].isin(filter_values)
-                ]
+                new_reverse_table_columns.append(column)
+        reverse_table.columns = new_reverse_table_columns
+        standard_direction_table = pd.concat(
+            [standard_schema_table.loc[~reverse_mask, :], reverse_table]
+        )
+        return standard_direction_table
+
+    def _standardize_organization_name(
+        self, standard_schema_table: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Convert any specified organization names to full_name and transactor_type organization"""
+        organization_name_columns = [
```

Comment:
nit: `standard_schema_table.columns[standard_schema_table.columns.str.endswith("organization_name")]` is (sort of) more concise, though fine if you prefer this way.